src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -1,7 +1,12 @@
 package core.basesyntax;
 
+import java.util.Random;
+
 public class ColorSupplier {
+    private final Random random = new Random();
+
     public String getRandomColor() {
-        return null;
+        int index = new Random().nextInt(Color.values().length);
```

Comment:
Pay attention on the comment from Andriy

Reply:
removed this variable ✔️

Reply:
You don't need to remove this variable.
In your previous version you declare the field 
private final Random random = new Random();

but not use this variable in method

int index = new Random().nextInt(Color.values().length);

So the correct version should be

```
private final Random random = new Random();

public String getRandomColor() {
        int index = random.nextInt(Color.values().length);
        return Color.values()[index].name().toLowerCase();
}

```

Please look at common mistakes
[Think about which variables should be local in the method and which should be class-level](https://mate-academy.github.io/jv-program-common-mistakes/java-core/oop/lottery#think-about-which-variables-should-be-local-in-the-method-and-which-should-be-class-level)